strongbox-security/strongbox-user-management/src/main/java/org/carlspring/strongbox/users/userdetails/StrongboxUserManager.java

```diff
@@ -0,0 +1,15 @@
+package org.carlspring.strongbox.users.userdetails;
+
+import org.carlspring.strongbox.users.dto.User;
+import org.springframework.security.core.userdetails.UserDetails;
+
+public interface StrongboxUserManager
+{
+
+    User findByUserName(String username);
```

Comment:
Please, rename this to `findByUsername`. Thanks! :)

Reply:
fixed

Reply:
Thanks!